cmd/gossamer/flags.go

```diff
@@ -171,32 +176,21 @@ var (
 	}
 )
 
+// flag sets that are shared by multiple commands
 var (
 	// GlobalFlags are flags that are valid for use with all commands
 	GlobalFlags = []cli.Flag{
 		VerbosityFlag,
+		ForceFlag,
```

Comment:
hmm, not sure if this is global, since it's not valid with just `./bin/gossamer`

Reply:
Not at the moment but I figured it was a global flag in the sense that we use it whenever there is a confirm prompt even though we are not currently using it in the root command.

Reply:
I can move it but it seems global to me. Where would you recommend I move it to?

Reply:
I think we should only make it global when necessary, if you could move it to InitFlags and ExportFlags that'd be good